src/Tree.hpp

```diff
@@ -58,6 +58,8 @@ namespace RRT
         T _state;
         std::list<Node<T> *> _children;
         Node<T> *_parent;
+        int error = 50;
```

Comment:
General robocup convention is to name private instance variables beginning with an underscore.